config/intellij-idea-inspections.xml

```diff
@@ -4638,7 +4638,8 @@
     ,com.puppycrawl.tools.checkstyle.internal.TestUtil,assert.*, ,
     ,nl.jqno.equalsverifier.EqualsVerifier,verify.*, ,
     ,org.checkstyle.base.AbstractItModuleTestSupport,verify.*, ,
-    ,com.tngtech.archunit.lang.ArchRule,check"/>
+    ,com.tngtech.archunit.lang.ArchRule,check ,
+    ,com.puppycrawl.tools.checkstyle.filters.SuppressionFilterTest, assert.*"/>
```

Comment:
`package.name, methodnamePattern`

Reply:
yes. I had figured that. pushing changes.

Reply:
I have made some changes. Please check. CI is still showing error.

Reply:
Done. Made some changes.

Reply:
@nrmancuso ``run-inspections`` is failing. Seems like the suppression I created didn't work.

Reply:
@shubh220922 , please download IDEA ,
follow https://checkstyle.org/idea.html#Inspections and play with config setup to make it be suppressed.

Reply:
Done